src/ashish/A9_ShopSystem.java

```diff
@@ -0,0 +1,75 @@
+package ashish;
+
+public class A9_ShopSystem {
+
+	int stockMaggie, stockDosa, stockPouches, stockPanipuri, stockMasala;
+	
+	void setAvailableQuantity(int maggie, int dosa, int pouches, int panipuri, int masala) {
+		stockMaggie = maggie;
+		stockDosa = dosa;
+		stockPouches = pouches;
+		stockPanipuri = panipuri;
+		stockMasala = masala;
+	}
+	
+	void purchaseItem(int maggie, int dosa, int pouches, int panipuri, int masala) {
+		stockMaggie = stockMaggie - maggie;
```

Comment:
1. we should also check that do we have sufficient maggie for request order or not.

Reply:
sure @KrishnaKTechnocredits , Thank you for correction. I will push suggested changes along with Assignment_10 today.